installation/.files/google.1

```diff
@@ -17,6 +17,11 @@ instead of
 .B I lost the Game
 , like that you will find where someone lost and you will lose too.
 .TP
+\fBPlus:\fR
```

Comment:
Adding the plus is a good idea !
And thanks for correcting the typo.
BUT: I've put a lot of creativity in creating original content for each example. Reusing "Carmen Electra" is not the best way to put your creative capabilities into this document :)